tests/testAutopas/testingHelpers/GridGenerator.h

```diff
@@ -0,0 +1,78 @@
+/**
+ * GridGenerator.h
+ *
+ *  Created on: 5/25/18
+ *     Aauthor: F. Gratl
+ */
+
+#pragma once
+#include "AutoPas.h"
+
+template<class Particle, class ParticleCell>
```

Comment:
if you put the templates to the functions (with the respective defaultParticle (see below)) then you don't have to write as much when calling the functions (no templates needed)

Reply:
-> 39969ff48b9df5d6f3dafe274b58823aa93ab758

Definition of Particle is still required when passing a cell vector:
`GridGenerator::fillWithParticles<SomeParticle>(cells, edgeLengths);`

Reply:
> not if you pass a `Particle defaultParticle` as argument to the function, then you can also add a few other things, like default parameters

-> 368fb46b7b6cdfc77b1ad871bb932c3eef01024b